docs/document/docs/getting-started/custom-css.md

```diff
@@ -0,0 +1,75 @@
+---
+id: custom-css
+title: Custom CSS
+sidebar_label: Custom CSS
+---
+
+## Overview
+
+**Kintone UI Component (KUC)** provides some Custom CSS properties for each component by using [CSS inheritance](#css-inheritance) and [CSS variables and custom properties](#css-variables-and-custom-properties) together. It's easy to create customizable elements. By applying CSS selectors to customize CSS custom properties, tree-based and per-instance theming are straightforward to apply.
+
+### CSS inheritance
+
+CSS inheritance lets parent and host elements propagate certain CSS properties to their descendants.
+
+### CSS variables and custom properties
+
+All CSS custom properties (`--custom-property-name`) inherit. You can use this to make your component's styles configurable from outside.
+
+KUC will publish some properties for each element, based on that users can customize CSS easily.
+
+## Usage
+
+You can set the value for `--my-background` by using the CSS element selector:
+
+```javascript
+<style>
+  my-element {
+    --my-background: #d9092f;
+  }
+</style>
+<my-element></my-element>
+```
+
+`--my-background` is configurable per instance of `my-element`:
+
+```javascript
+<style>
+  my-element {
+    --my-background: #d9092f;
+  }
+  my-element.sample-class {
+    --my-background: #a8a632;
+  }
+   my-element#sample-id {
+    --my-background: #1ac9b8;
+  }
```

Comment:
You are using "my-element" before class name/id here.
And on Sample Code of Text, you are directly using class name/id as element selector.
I think it's better to unify it.